intel-sgx/enclave-runner/src/usercalls/mod.rs

```diff
@@ -38,9 +40,11 @@ use ipc_queue::{self, DescriptorGuard, Identified, QueueEvent};
 use sgxs::loader::Tcs as SgxsTcs;
 
 use crate::loader::{EnclavePanic, ErasedTcs};
-use crate::tcs::{self, CoResult, ThreadResult};
+use crate::tcs::{self, CoResult, ThreadResult, EXITING};
 use self::abi::dispatch;
 use self::interface::{Handler, OutputBuffer};
+use crate::tcs::has_vdso_sgx_enter_enclave;
```

Comment:
Nitpick: Usually we don't include functions but refer to them as `tcs::has_vdso_sgx_enter_enclave` (also having two separate use statement for the same module on disjunct lines is a bit weird)